tests/utils.py

```diff
@@ -93,17 +94,25 @@ def assert_allclose(
     np.testing.assert_allclose(actual, desired, *args, **kwargs)
 
 
-def test_pretrained_networks(network, input_param, device):
+@contextmanager
+def skip_if_downloading_fail():
     try:
+        yield
+    except (ContentTooShortError, HTTPError, ConnectionError) as e:
+        raise unittest.SkipTest(f"error while downloading: {e}") from e
+    except RuntimeError as rt_e:
+        if "network issue" in str(rt_e):
+            raise unittest.SkipTest(f"error while downloading: {rt_e}") from rt_e
+        if "gdown dependency" in str(rt_e):  # no gdown installed
+            raise unittest.SkipTest(f"error while downloading: {rt_e}") from rt_e
```

Comment:
Will this `rt_e` error explicitly say something about installing `gdown`?

Thanks.

Reply:
yes it's from this https://github.com/Project-MONAI/MONAI/blob/b1a96c571fa9ec3dccf7e16a0105df78d6f00889/monai/apps/utils.py#L191